org-code-javabuilder/lib/src/main/java/org/code/javabuilder/Util.java

```diff
@@ -15,7 +15,11 @@
 /** A set of static utility functions that are used in multiple locations */
 public class Util {
   private static final String[] JAR_NAMES = {
-    "neighborhood-full.jar", "theater-full.jar", "playground-full.jar", "studentlib-full.jar"
+    "neighborhood-full.jar",
+    "theater-full.jar",
+    "playground-full.jar",
+    "studentlib-full.jar",
+    "validation-full.jar"
```

Comment:
(May be outside the scope of this change) I remember there was some discussion on only including validation when running test code. Would we want to selectively include the jar?

Reply:
I was planning to handle this as part of the class loader. Currently no one can actually use an org.code.validation package because it's not in the allow list (they can import it, but that won't help much)